jobs/cjd-sample/quickstart/main.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START quick_start]
+package main
```

Comment:
newline before package, otherwise this ends up as the package-level comment.

Reply:
Done